### PR TITLE
adjust internal documentation according to #7812

### PR DESCRIPTION
**Pull request by cebe**
adjust internal documentation according to #7812 

Versioning was not up to date with what was proposed in the issue and I also included project organization.

This is WIP, so do not merge yet. I am planning to add more things on release automation soon.
We also need a branching/merging concept that allows for making bugfix-only releases.

### CODE REVIEW

**Comment by cebe** [comment]
> We also need a branching/merging concept that allows for making bugfix-only releases.

I tend to reject this idea and do normal patch releases as we are already doing but speed things up by making release more automated. Tagging on one branch is easy, separating bug and feature releases will require a lot of back and forth merging which is not easily handlealbe.

@yiisoft/core-developers opinions?

File: docs/internals/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+----------------------
+
+- [How to Report an Issue](report-an-issue.md)
+- [Getting started](getting-started.md)
+- [Git workflow for Yii 2 contributors](git-workflow.md) - a step by step guide on how to set up your dev environment and start contribute to Yii.
+- [Yii2 Core Framework Code Style](core-code-style.md)
+- [Yii2 View Code Style](view-code-style.md)
+
+
+Dokumentation
```
Comment (klimov-paul) [inline docs/internals/README.md]: "Dokumentation" -> "Documentation"

File: docs/internals/versions.md [hunk 2]
```diff
@@ -9,41 +9,77 @@ Within the core developer team, we have emphasized several times that it is impo
...
-* Require pre-releases: `2.X.0-alpha`, `2.X.0-beta`, `2.X.0-rc`
-* Requires major news releases and marketing effort.
 
 
-## Major Releases `X.0.0`
+## Releases
 
-It's like 2.0 over 1.0. We expect this only happens every 3 to 5 years, depending on external technology advancement
-(such as PHP upgraded from 5.0 to 5.4).
+Both Yii2  Framework and official extension projects follow the above versioning and branching policies.
```
Comment (SilverFire) [inline docs/internals/versions.md]: `Yii2  Framework` - extra space

**Comment by SilverFire** [comment]
:+1:

File: docs/internals/automation.md [hunk 3]
```diff
@@ -13,3 +13,8 @@ There are some tasks that are done automatically when working on Yii:
   Run `./build/build php-doc/fix` to fix them.
   Check the changes before you commit them as there may be unwanted changes because the command is not perfect.
   You may use `git add -p` to review the changes.
+
+- Updating the Mime Type Magic file (`framework/helpers/mimeTypes.php`) from the Apache HTTPd repository.
+  Run `./build/build mime-type` to update the file.
+
+All of the above commands are included in the [release process](). They may be run between releases but it is not necessary.
```
Comment (samdark) [inline docs/internals/automation.md]: Is it to be developed later?

**Comment by samdark** [comment]
:+1:

File: docs/internals/automation.md (hunk 3 above)
Comment (cebe) [inline docs/internals/automation.md]: I have completed the release command while doing the 2.0.8 release, will merge this soon.

**Pull request closed without merging**